keras_cv/layers/preprocessing/rgb_shift_test.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing.rgb_shift import RGBShift
+
+
+class RGBShiftTest(tf.test.TestCase):
+    def test_return_shapes(self):
```

Comment:
lets implement a value test that actually inspects the return values.

This can be done by passing `factor=(0.5, 0.5)` which will then make get_random_transformation always return 0.5 for the transforms, so you can know what it will do.  Try this for 0.5, maybe 0.7

Reply:
Done.